### ISSUE TITLE
require-computed-property-dependencies false positive

### ISSUE THREAD

**Issue by boris-petrov** [ember-cli/eslint-plugin-ember#475]
Using string interpolation like so:

```javascript
computed(`${dependentKey}.@each.${propertyKey}`, function() {
```

Causes `require-computed-property-dependencies` to be triggered which is wrong.

P.S. Actually it is not "wrong" but I guess outputting the warning in such a case is pointless - obviously dynamic stuff is happening and the linter can not check it.
